bots/tests-scan

```diff
@@ -440,7 +433,7 @@ def cockpit_tasks(api, update, contexts, repo, pull_data, pull_number, sha, amqp
             if not update or update_status(api, revision, context, status, changes):
                 results.append((priority, "pull-%d" % number, number, revision, "pull/%d/head" % number, context, base, repo, None))
 
-        # if a cockpit (default repo) PR changes bots/, also trigger all external tests against this cockpit PR
+        # if a cockpit repo PR changes bots/, also trigger all external tests against this cockpit PR
 
         def trigger_externals():
             if repo:  # already a non-cockpit project
```

Comment:
In spirit of the change above, I think this should also be made explicit with `if repo != "cockpit-project/cockpit"`.